src/components/Header/index.js

```diff
@@ -13,25 +11,18 @@ import {
   Nav,
   NavItem } from 'reactstrap';
 
-class Header extends Component {
-
-  render() {
-    return (
-      <div>
-        <Navbar color="faded" light expand="md">
-          <NavbarBrand href="/">TradeHub</NavbarBrand>
-          <NavbarToggler onClick={this.toggle} />
-          <Collapse isOpen={true} navbar>
-            <Nav className="ml-auto" navbar>
-              <NavItem>
-                <NavLink to="/tradehub">TradeHub</NavLink>
-              </NavItem>
-            </Nav>
-          </Collapse>
-        </Navbar>
-      </div>
-    )
-  }
+export default class Header extends Component {
+  render = () => (
+    <Navbar color="faded" light expand="md">
+      <NavbarBrand href="/">TradeHub</NavbarBrand>
+      <NavbarToggler onClick={this.toggle} />
+      <Collapse isOpen navbar>
+        <Nav className="ml-auto" navbar>
+          <NavItem>
+            <NavLink to="/tradehub">TradeHub</NavLink>
+          </NavItem>
+        </Nav>
+      </Collapse>
+    </Navbar>
+  );
```

Comment:
Make this a functional component

Reply:
Navbar will be covered in another branche, I want to change and improve it.